ci/lib/templates.lib.txt

```diff
@@ -81,7 +81,7 @@ remote_shell cmake -E chdir (@= path @) ctest -C ${CMAKE_CONFIG} -j(@= str(paral
 (@ def dotnet_bash_task(path): -@)
 set -ueo pipefail
 (@= remote_functions() @)
-remote_shell chdir (@= path @) && dotnet test
+remote_shell cd (@= path @) && dotnet test
```

Comment:
Use the `cmake -E chdir` like all the other test executions. It normalizes the `cd` like operations across all platforms.